src/main/webapp/index.html

```diff
@@ -43,6 +43,10 @@
           <span class="sr-only">Loading...</span>
         </div>
       </div>
+      <div id="error-surfacer" style="display:none">
+        <h4 id="error-display-text"style="color:rgb(255, 99, 71);"> Something went wrong :( </h4>
+        <h5 id="error-details"style="color:rgb(255, 99, 71);"><h5>
+      </div>
```

Comment:
We might not want to show the error message - in some cases it might help, but the majority of video retrieval errors result in "Unexpected < token at position X" (or something like that). It may be easier to just ask the user to try another video.

Otherwise, if we wanted to give a specific error, maybe comments are turned off or comments are in a specific language, we could focus on specializing the error message on front end.